code_generator/GdkCodeGenerator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -287,15 +310,91 @@ namespace <#= qualifiedNamespace #>
+                    data.Requests = new List<<#= wrappedCommandRequestType #>>();
+                    data.Requests.Add(new <#= wrappedCommandRequestType #>(op.RequestId.Id,
+                        op.CallerWorkerId,
+                        op.CallerAttributeSet,
+                        deserializedRequest));
+
+                    entityManager.AddComponentData(entity, data);
+                }
             }
 
-            private void On<#= command.CommandName #>Response(CommandResponseOp op)
+            private void On<#= commandDetails.CommandName #>Response(CommandResponseOp op)
             {
-                // TODO: Implement commands
+                if (!<#= commandStorage #>.CommandRequestsInFlight.TryGetValue(op.RequestId.Id, out var requestBundle))
+                {
+                    throw new InvalidOperationException($"Could not find corresponding request for RequestId {op.RequestId.Id} and command <#= commandDetails.CommandName #>.");
+                }
+
+                var entity = requestBundle.Entity;
+                if (!entityManager.Exists(entity))
+                {
+                    LogDispatcher.HandleLog(LogType.Error, new LogEvent(EntityNotFound)
+                        .WithField(LoggingUtils.LoggerName, LoggerName)
+                        .WithField("Op", "CommandResponseOp - <#= commandDetails.CommandName #>")
+                        .WithField("Component", "<#= componentDetails.TypeName #>")
+                    );
+                    return;
+                }
+
+                <#= commandDetails.ResponseType #>? response = null;
+                if (op.StatusCode == StatusCode.Success)
+                {
+                    response = <#= commandDetails.ResponseType #>.Serialization.Deserialize(op.Response.SchemaData.Value.GetObject());
+                }
+
+                if (entityManager.HasComponent<<#= commandResponseBufferType #>>(entity))
+                {
+                    var data = entityManager.GetComponentData<<#= commandResponseBufferType #>>(entity);
+                    data.Responses.Add(new <#= wrappedCommandResponseType #>(op.EntityId,
+                        op.Message,
+                        op.StatusCode,
+                        response,
+                        requestBundle.Request));
+                }
+                else
+                {
+                    var data = new <#= commandResponseBufferType #>();
+                    data.CommandListHandle = <#= qualifiedNamespace #>.<#= componentDetails.ComponentName #>.ReferenceTypeProviders.<#= commandDetails.CommandName #>ResponsesProvider.Allocate(World);
+                    data.Responses = new List<<#= wrappedCommandResponseType #>>();
+                    data.Responses.Add(new <#= wrappedCommandResponseType #>(op.EntityId,
```

Comment:
duplicate from above

Reply:
See events PR on how this was done there https://github.com/spatialos/UnityGDK/pull/208/files#diff-e0ac405a3c4d8443d366a7c5462693afR154